docs/docs/development/build-flash.md

```diff
@@ -90,7 +90,11 @@ This produces `left` and `right` subfolders under the `build` directory and two
 
 ### Building from `zmk-config` Folder
 
-Instead of building .uf2 files using the default keymap and config files, you can build directly from your [`zmk-config` folder](../user-setup.md#github-repo) by adding
+Getting your `zmk-config` to be used by your build environment differs based on what type of build environment you are using.
+
+#### Local/Host Build Environment.
+
+Instead of building .uf2 files using the default keymap and config files, you can build directly from your [`zmk-config` folder](../user-setup#github-repo) by adding
```

Comment:
```suggestion
Instead of building .uf2 files using the default keymap and config files, you can build directly from your [`zmk-config` folder](../user-setup.md#github-repo) by adding
```
I think this was changed by a PR by @dxmh, so the diff should be cleaner with the `.md` added back.